pkg/asset/installconfig/powervs/validation.go

```diff
@@ -25,6 +25,12 @@ func Validate(config *types.InstallConfig) error {
 			fldPath := field.NewPath("compute").Index(idx)
 			allErrs = append(allErrs, validateMachinePool(fldPath, &compute)...)
 		}
+		//Check whether Machine Pool CIDR is /24 by checking whether its suffix is "/24".
+		if !(strings.HasSuffix((&config.Networking.MachineNetwork[0].CIDR).String(),"/24")){
```

Comment:
Definitely iterate over all MachineNetworks. We don't handle more than one currently but will be more robust in case we ever decide to.